Eastern-Research-Group/Trade-off_Tool_for_Sampling
# Feature/53 save calculate results settings

## Related Issues:
* [TOTS-53](https://ergcloud.atlassian.net/browse/TOTS-53)

## Main Changes:
* Added calculate settings to the output of the publishing logic.
  * This involved making the settings on the calculate resources tab dependent on the selected sampling plan.

## Steps To Test:
1. Create a sampling plan
2. Add some samples to it
3. Create another sampling plan 
4. Add some samples to it
5. Go to the Calculate Resources tab
6. Change some of the settings
7. Go to Create Plan tab and back to Calculate Resources tab
8. Verify the settings you entered are still there
9. Switch to the other sampling plan you created
10. Verify the settings on the Calculate Resources tab are now the defaults
11. Change the settings on the Calculate Resources tab
12. Publish the output
13. In AGO, verify your calculate settings are in the feature service you just published
14. Back in TOTS, click "Start Over" on the "Create Plan" tab
15. From the "Add Data" tab, add in the layer you just published
16. Go to "Create Plan" tab and select the plan
17. Go to "Calculate Resources" 
18. Verify the settings match what was saved
